packages/storybook/stories/va-alert-uswds.stories.tsx

```diff
@@ -240,56 +235,6 @@ Default.args = {
 };
 Default.argTypes = propStructure(alertDocs);
 
-export const SignInOrToolPrompt = Template.bind(null);
-SignInOrToolPrompt.args = {
-  ...defaultArgs,
-  children: (
-    <>
-      <p class="vads-u-margin-y--0">
-        You can use our new mobile app to check the status of your claims or
-        appeals on your mobile device. Download the{' '}
-        <strong>VA: Health and Benefits</strong> mobile app to get started.
-      </p>
-      <va-button primary-alternate text="Sign-in to VA.gov" />
-    </>
```

Comment:
These aren't deleted. I just moved them down with the new story that was added because it seemed to make more sense to have them grouped there.